pilot/pkg/model/config.go

```diff
@@ -526,14 +526,15 @@ var (
 		Validate:      ValidateServiceRoleBinding,
 	}
 
-	// RbacConfig describes an global RBAC config.
+	// RbacConfig describes the mesh level RBAC config.
 	RbacConfig = ProtoSchema{
-		Type:        "rbac-config",
-		Plural:      "rbac-configs",
-		Group:       "config",
-		Version:     istioAPIVersion,
-		MessageName: "istio.rbac.v1alpha1.RbacConfig",
-		Validate:    ValidateRbacConfig,
+		ClusterScoped: true,
```

Comment:
Have you try 'upgrade' Istio from previous version to this? As you already define this CRD in previous release, you might not be able to change the scoped, without remove the old one. I'm not sure what the best way for migration, May be add smt to the installation template to remove old CRD (assuming no one using this yet) before (re)install Istio.

Reply:
The way I 'upgrade' Istio is simply remove the existing one and then install the newer version. And I think there is no one really using this CRD, even in test. I didn't change other fields but only the ClusterScoped, is this considered unsafe?

Reply:
Thanks for the remind, Diem. I'll test it again and probably add some notes to the doc, better to remove the old RbacConfig when upgrading, but need to investigate how to do this.